datagovharvester/utils/s3_utilities.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+import botocore
+
+
+def create_s3_client(s3_config):
+    S3 = None
+    error_message = None
+    try:
+        S3 = boto3.client("s3", **s3_config)
+    except botocore.exceptions.ClientError as e:
+        error_message = e
+
+    return S3, error_message
+
+
+def create_s3_bucket(S3, bucket_name):
+    error_message = None
+
+    try:
+        S3.create_bucket(Bucket=bucket_name)
+    except Exception as e:
+        error_message = e
+
+    return bucket_name, error_message
```

Comment:
This is an unlikely function. Can probably just be run wholesale in the tests, and not implemented here.

Reply:
removed the function and moved bucket creation [here](https://github.com/GSA/datagov-harvesting-logic/blob/54226c79f9e595315b8f4d3e961eb643628be08e/tests/extract/test_dcatus.py#L15)